neqo-http3/src/settings.rs

```diff
@@ -21,6 +21,10 @@ const SETTINGS_MAX_HEADER_LIST_SIZE: SettingsType = 0x6;
 const SETTINGS_QPACK_MAX_TABLE_CAPACITY: SettingsType = 0x1;
 const SETTINGS_QPACK_BLOCKED_STREAMS: SettingsType = 0x7;
 const SETTINGS_ENABLE_WEB_TRANSPORT: SettingsType = 0x2b60_3742;
+// draft-ietf-masque-h3-datagram-04.
+// We use this old value because the current web-platform test only supports
+// this value.
+const SETTINGS_H3_DATAGRAM_DRAFT04: SettingsType = 0x00ff_d277;
```

Comment:
RFC 9297 has been published now.  Where do we stand on that?

```suggestion
const SETTINGS_H3_DATAGRAM: SettingsType = 0x33;
```

Reply:
For sure we want to use the value defined in RFC.
I'll change this.

Reply:
Sounds reasonable.  We could also send both (as client) and just pick one (as server).

Reply:
Thanks! I think sending both is easier.